avro-codegen/build.gradle

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright 2018 LinkedIn Corp.
- * Licensed under the BSD 2-Clause License (the "License"). 
+ * Licensed under the BSD 2-Clause License (the "License").
```

Comment:
What happened here? IDE going rogue on you? There's lots of these trivial changes in your PR...?

Reply:
some non-printable character was giving the build issues